src/components/Header/Header.story.jsx

```diff
@@ -138,7 +138,10 @@ const headerPanel = {
 
 storiesOf('Watson IoT|Header', module)
   .add('Header action buttons with dropdowns', () => (
-    <StyledHeader {...HeaderProps} headerPanel={headerPanel} />
+    <div style={{ width: '100%', height: '100vh' }}>
+      <StyledHeader {...HeaderProps} headerPanel={headerPanel} />
+      <div id="skip" />
```

Comment:
What's this div for?

Reply:
this is just for the story. You need to have an actual id to skip to in order to pass a11y scan